RealmSwift/Tests/RealmCollectionTypeTests.swift

```diff
@@ -522,6 +522,13 @@ class RealmCollectionTypeTests: TestCase {
         XCTAssertTrue(collection.freeze().isFrozen)
     }
 
+    func testThaw() {
+        let frozen = collection.freeze()
+        XCTAssertTrue(frozen.isFrozen)
+        let thawed = frozen.thaw()
+        XCTAssertFalse(thawed.isFrozen)
```

Comment:
This test should assert that that frozen collection cannot be mutating and that the thawed collection can.